tools/c7n_gcp/tests/test_loadbalancer.py

```diff
@@ -239,6 +240,37 @@ def test_loadbalancer_backend_bucket_get(self):
         self.assertEqual(instance['kind'], 'compute#backendBucket')
         self.assertEqual(instance['name'], 'newbucket')
 
+    def test_loadbalancer_backend_bucket_delete(self):
+        project_id = 'custodian-test-project-0'
+        session_factory = self.replay_flight_data('lb-backend-buckets-delete',
+                                                  project_id=project_id)
+        base_policy = {'name': 'lb-addresses-delete',
+                       'resource': 'gcp.loadbalancer-backend-bucket'}
+
+        policy = self.load_policy(
+            dict(base_policy,
+                 filters=[{'type': 'value',
+                           'key': 'bucketName',
+                           'op': 'eq',
+                           'value': 'custodian-bucket-0'}],
+                 actions=[{'type': 'delete'}]),
+            session_factory=session_factory)
+        resources = policy.run()
+        self.assertEqual(2, len(resources))
+        self.assertEqual('custodian-bucket-0', resources[0]['bucketName'])
+        self.assertEqual('custodian-bucket-0', resources[1]['bucketName'])
+        self.assertEqual('custodian-backend-bucket-1', resources[0]['name'])
+        self.assertEqual('custodian-backend-bucket-3', resources[1]['name'])
+
+        if self.recording:
+            sleep(5)
+
+        policy = self.load_policy(base_policy, session_factory=session_factory)
```

Comment:
don't use policies to verify policies, just use a client call

Reply:
Updated